testsuite/smoke-test/src/key_manager.rs

```diff
@@ -54,6 +54,9 @@ fn test_key_manager_consensus_rotation() {
         ChainId::test(),
     );
 
+    // Add some time padding to ensure the libra timestamp increases on-chain
+    sleep(Duration::from_secs(10));
```

Comment:
:(. Isn't 10 seconds pretty long?

Reply:
Yeah, this is a beefy buffer.. I was trying to be conservative to reduce the flakes around the timestamp. As discussed offline, let's leave it as-is for now, and I'll monitor flakiness.... if things improve, I can reduce the timeout 😄